tests/unit/services/traction/QcResultsUpload.spec.js

```diff
@@ -0,0 +1,108 @@
+import { createQcResultsUploadResource } from '@/services/traction/QcResultsUpload'
+
+describe('QcResultsUpload', () => {
+  describe('#createQcResultsUploadResource', () => {
+    // Setup some of the parameters we'll be testing with
+    let csv = `,,SAMPLE INFORMATION,,,,,,,,,,,,,VOUCHERING,,,,EXTRACTION/QC,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,COLUMN JUST FOR TOL,COLUMN JUST FOR TOL,SE LIMS,
```

Comment:
maybe extract this and import an actual csv in tests/data/csv

Reply:
Done